packages/rest-hooks/jest.config.js

```diff
@@ -7,17 +10,21 @@ module.exports = {
   globals: {
     'ts-jest': {
       babelConfig: 'src/.babelrc',
+      tsConfig: 'src/tsconfig.json',
     },
   },
   transform: {
     '^.+\\.tsx?$': 'ts-jest',
     '^.+\\.jsx?$': 'babel-jest',
   },
   testRegex: '(/__tests__/.*|(\\.|/)(test|spec))\\.tsx?$',
-  coveragePathIgnorePatterns: ['test', 'react-integration/hooks/useSelection'],
+  coveragePathIgnorePatterns: ['react-integration/hooks/useSelection'],
   moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json'],
   moduleNameMapper: {
-    '^~/(.*)$': '<rootDir>/src/$1',
+    ...pathsToModuleNameMapper(compilerOptions.paths, {
```

Comment:
this DRY's up the concept of ~/ to only need to be in the tsconfig and then it's copied here.